mindjunkies/courses/tests/test_views.py

```
@@ -91,6 +92,7 @@ def test_create_course_get_pending_token(client, teacher_user):
 
 @pytest.mark.django_db
 def test_create_course_post_success(client, teacher_user, category):
+    course = "courses.Course"
```

Ruff (0.8.2)

Local variable `course` is assigned to but never used

Remove assignment to unused variable `course`

(F841)